arviz/plots/backends/bokeh/violinplot.py

```diff
@@ -71,8 +71,8 @@ def plot_violin(
             rug_x = -np.abs(np.random.normal(scale=max(dens) / 3.5, size=len(val)))
             ax_.scatter(rug_x, val, **rug_kwargs)
 
-        per = np.percentile(val, [25, 75, 50])
-        hdi_probs = hdi(val, hdi_prob, multimodal=False)
+        per = np.percentile(val[np.isfinite(val)], [25, 75, 50])
+        hdi_probs = hdi(val[np.isfinite(val)], hdi_prob, multimodal=False)
```

Comment:
this should be `hdi_probs = hdi(val, hdi_prob, multimodal=False, skipnan=True)`

Reply:
I think this is still failing. See:
```
val = np.r_[np.random.normal(size=50), np.nan]
az.hdi(val, skipnan=True)

array([-1.77298977,         nan])
```

Reply:
Which is reflected here in the original example
![image](https://user-images.githubusercontent.com/25507629/92742896-6fffe600-f356-11ea-83c3-a519cee24f3b.png)